Lib/fontParts/objects/base/validators.py

```diff
@@ -115,10 +115,11 @@ def validateLayerName(value):
 # -----
 
 def validateGlyphName(value):
-    """
-    XXX implement
-    """
-    return value
+    if not isinstance(value, basestring):
+        raise FontPartsError("Glyph names must be unicode strings, not %s." % type(value).__name__)
```

Comment:
No problem. I need to document what the validators need to do and what they should return. Thanks for doing this!